### DISCUSSION TITLE
Question about negative edges using `LinkNeighborLoader`

### DISCUSSION THREAD

**Discussion by ogawayuto** [pyg-team/pytorch_geometric#8279] [Q&A] [answered]
Hi, I'm working on inductive link prediction task on a single large graph and i have a quesiton about `LinkNeighborLoader` function. 

**Situation**
In each epoch, I have fixed positive/negative edges in `edge_index` and they're hold as edges with label 0 (while positve ones are label 1). I'm applying `LinkNeighborLoader` to this graph. 

**What I want to do**
I want returned subgraphs to have only positive edges used as message passing edges while negative edges are not used for it but used as seed edges. 
Does anyone have an idea to implement this?

**Comment by rusty1s** [comment, marked as answer]
This is what the `LinkNeighborLoader` gives you by default. `data.edge_index` is used for message passing (the edges that exist), while `edge_label_index` denotes the seed edges.

**Reply by ogawayuto** [reply]
Thank you rusty! I was confusing `edge_index` and `edge_label_index`.
